telemetry/definitions/commonDefinitions.json

```diff
@@ -3294,6 +3438,419 @@
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_logApiLatency",
+            "passive": true,
+            "description": "A general logging measure for all client side API's latency.",
+            "metadata": [
+                {
+                    "type": "codeTransformApiNames",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRunTimeLatency",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRequestId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformUploadId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformTotalByteSize",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_logGeneralError",
+            "passive": true,
+            "description": "A general logging measure for all client side API errors.",
+            "metadata": [
+                {
+                    "type": "codeTransformApiErrorMessage",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_logApiError",
+            "passive": true,
+            "description": "A general logging measure for all client side API errors.",
+            "metadata": [
+                {
+                    "type": "codeTransformApiErrorMessage",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformApiNames",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRequestId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
```

Comment:
Is this used?

Reply:
Yes. This is also a "new" metric added in the PR's -> codeTransform_dependenciesCopied
https://github.com/aws/aws-toolkit-jetbrains/pull/4030/